packages/vhd-lib/merge.js

```diff
@@ -56,16 +57,19 @@ module.exports = limitConcurrency(2)(async function merge(
       flags: 'r+',
       checkSecondFooter: mergeState === undefined,
     })
-    const childVhd = yield openVhd(childHandler, childPath)
+    const childVhd = yield VhdSynthetic.open(childHandler, childPath)
```

Comment:
Add a comment above the function:

```js
// childPath can be array to create a synthetic VHD from multiple VHDs
```

```suggestion
    const childVhd = yield Array.isArray(childPath) ? VhdSynthetic.open(childHandler, childPath) : openVhd(childHandler, childPath)
```